src/containers/Secrets/Secrets.js

```diff
@@ -226,6 +226,8 @@ export /* istanbul ignore next */ class Secrets extends Component {
         });
       });
       const serviceAccountsString = serviceAccountsWithSecret.join(', ');
+      const secretAsBase64 = Buffer.from(secret.username, 'base64');
```

Comment:
`Buffer` is not a standard browser type, it'll work in unit tests as it's provided by Node.js but it's unlikely to work in many browsers.

There's a well-supported (IE10+) browser native API for decoding base64 strings: `window.atob`

`const secretUsernameToDisplay = atob(secret.username);`

Reply:
Excellent, thanks I'll give that a go

Reply:
Done